sdk/tests/js/tests/gl-object-get-calls.js

```diff
@@ -865,7 +865,47 @@ if (contextVersion > 1) {
 	    return gl.getQueryParameter(query, pname);
 	}
     );
+
+    debug("");
+    debug("Test getFragDataLocation");
+    var baseVertShader = '' +
+    '#version 300 es\n' +
+    'uniform vec4 normal;\n' +
+    'in vec4 vertext;\n' +
+    'out vec4 position;\n' +
+    'void main (void)\n' +
+    '{\n' +
+    '       position = vec4(0.0);\n' +
+    '       gl_Position = vertext;\n' +
+    '}\n';
+    var baseFragShader = '' +
+    '#version 300 es\n' +
+    'layout(location = 0) in lowp vec4 position;\n' +
+    'layout(location = 1) out mediump vec4 fragColor;\n' +
+    'void main (void)\n' +
+    '{\n' +
+    '       fragColor = position;\n' +
+    '}\n';
+    var vertShader = gl.createShader(gl.VERTEX_SHADER);
+    gl.shaderSource(vertShader, baseVertShader);
+    gl.compileShader(vertShader);
+    shouldBe('gl.getShaderParameter(vertShader, gl.COMPILE_STATUS);', 'true');
+    var fragShader = gl.createShader(gl.FRAGMENT_SHADER);
+    gl.shaderSource(fragShader, baseFragShader);
+    gl.compileShader(fragShader);
+    shouldBe('gl.getShaderParameter(fragShader, gl.COMPILE_STATUS);', 'true');
+    var program = gl.createProgram();
+    gl.attachShader(program, vertShader);
+    gl.attachShader(program, fragShader);
+    gl.linkProgram(program);
+    shouldBe('gl.getProgramParameter(program, gl.LINK_STATUS)','true');
+    shouldBe('gl.getFragDataLocation(program, "vertextColor")', '-1');
+    shouldBe('gl.getFragDataLocation(program, "normal;")', '-1');
+    shouldBeNonZero('gl.getFragDataLocation(program, "vertext")');
+    shouldBe('gl.getFragDataLocation(program, "position")', '0');
```

Comment:
Why is this the case? position is an input variable to the fragment shader, not an output variable. It should be -1, no?